chrome/content/zotero/xpcom/sync/syncRunner.js

```diff
@@ -1244,20 +1259,23 @@ Zotero.Sync.Runner_Module = function (options = {}) {
 	
 	
 	var _getAPIKeyFromLogin = Zotero.Promise.coroutine(function* () {
-		var apiKey = "";
+		var apiKey;
 		let username = Zotero.Prefs.get('sync.server.username');
 		if (username) {
-			let password = Zotero.Sync.Data.Local.getLegacyPassword(username);
+			let password = Zotero.Sync.Data.Local.password;
```

Comment:
This is left over from an earlier revision, right? There's no such property anymore?